core/src/test/scala/org/typelevel/paiges/PaigesScalacheckTest.scala

```diff
@@ -168,42 +168,18 @@ class PaigesScalacheckTest extends FunSuite {
     }
   }
 
-  test("group law") {
```

Comment:
This test seems so specific to the non-Line implementation that I deleted it rather than figuring out what the new law was.  It fails as is.  We test other parts, such as the identities on line 175/6 below.